Assets/Scripts/Platform/PlatformColor.cs

```diff
@@ -27,36 +28,37 @@ private void Update()
 
             if (PlayerProperties.playerColor.Equals(platformColor) && !PlayerProperties.playerColor.Equals(playerColor))
```

Comment:
```suggestion
            if (PlayerProperties.playerColor.Equals(platformColor) && !PlayerProperties.playerColor.Equals(Color.white))
```
Táto farba bude vždy biela, čiže môžeš aj vymazať `playerColor` z atribútov a `Start`.